docs/mocksupport.rst

```diff
@@ -878,3 +878,64 @@ DMTF CIM schema download support
       :attributes:
 
    .. rubric:: Details
+
+
+.. _`Mock inmemory CIM repository`:
+
+Mock inmemory CIM repository
+----------------------------
+
+pywbem_mock provides a standard interface to a CIM repository that
+stores CIM classes, CIM instances, and CIM qualifier declarations.
```

Comment:
I think we should talk about the in-memory repo and not about the underlying standard repo interface. I suggest to change this sentence to:
```
The pywbem_mock module provides a CIM repository that
stores CIM classes, CIM instances, and CIM qualifier declarations
in memory.
```

Reply:
CLOSED - became moot since the last commit changed the module docstring.